chesster.js

```diff
@@ -51,6 +80,95 @@ var channels = {
         return "<#"+this.getId(name)+">";
     }
 };
+var channelsByID = { };
+
+
+var fuzzy_match = function (search_string, targets) {
+    if (!targets) { return null; } // TODO: not sure this is the right choice ...
```

Comment:
I would prefer to throw here. errors should be caught and logged in the handler.